Microsoft.O365.Security.Native.ETW/Microsoft.O365.Security.Native.ETW.NetCore.vcxproj

```diff
@@ -0,0 +1,198 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
let's update `.github/workflows/krabsetw.yml` to build on windows-2022 and also build the new projects